ws-server/src/server.rs

```diff
@@ -213,6 +249,24 @@ impl Builder {
 		self
 	}
 
+	/// Set a list of allowet `Origin` headers, connections comming in with a different
```

Comment:
```suggestion
	/// Set a list of allowed `Origin` headers, connections coming in with a different
```

Reply:
nitpick: to me "header" means the full `key: value` string, whereas here we are not setting the **header** but rather the set of hosts (with wildcards) that will be checked against the `Origin` headers sent in the client handshake.
So perhaps something like: "Set the list of allowed origins. During the handshake the `Origin` header of incoming connection requests will be checked against this list. Values should include the protocol. Wildcards are allowed, as per spec [here](link-to-spec)."

Reply:
There are no wildcards in the `Origin` headers (you can set one in `Access-Control-Allow-Origin`, but that doesn't apply to `WebSocket`. The only special `Origin` value is `null`, which the end user might pass as a string here, I don't think we need to build extra support for that here, and from what I've read the general wisdom is that you should never allow connections with `null` `Origin`, unless you are fine accepting it from any `Origin`.